src/ssl.cpp

```diff
@@ -17,12 +23,26 @@ using namespace std;
 #include "Globals.h"
 #include "ssl.h"
 
+#include <stdarg.h>
+#include <cstring>
+
+extern "C" {
+  extern double dnrm2_(int *, double *, int *);
```

Comment:
Could you add some comments to what these functions do? Just to make our lives a bit easier in the future.

Reply:
Added descriptions for all the functions.